site/router-topology/router-encapsulation.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Weave Router Encapsulation
+layout: default
+---
+
+
+When the Weave router forwards packets, the encapsulation looks
+something like this:
+
+    +-----------------------------------+
```

Comment:
I didn't think we had the time, but now we do. Yes, it would be great to replace those.  Should we use the one you did for the Docker University?